scripts/scripts.js

```diff
@@ -427,14 +433,14 @@ function ProblemIsUnchanged(buttonElement) {
 // CREATE NEW HISTORY ROW FROM CURRENT CONTENT
 // @param {mathContent} latex for new active area after being cleaned.
 // @return {nothing} No return value
-function NewMathEditorRow(mathContent) {
+function NewMathEditorRow(mathContent, showStepLabel, cleanup) {

```

Comment:
Do we need two flags here? Aren't they always opposites?

Reply:
Right